geometry_msgs/Pose pose
BoundingBox bounding_box

